integration/test/Test/Spar.hs

```diff
@@ -139,8 +142,7 @@ registerUser domain tid email = do
 activateEmail :: (HasCallStack, MakesValue domain) => domain -> String -> App ()
 activateEmail domain email = do
   (key, code) <- bindResponse (BrigInternal.getActivationCode domain email) $ \res -> do
-    res.status `shouldMatchInt` 200
-    key <- res.json %. "key" >>= asString
-    code <- res.json %. "code" >>= asString
-    pure (key, code)
+    (,)
+      <$> (res.json %. "key" >>= asString)
+      <*> (res.json %. "code" >>= asString)
```

Comment:
with this change you're losing the should-match-200 check.

Reply:
The 200 check does not provide much value, IMO.